odpf/guardian/v1beta1/guardian.proto

```diff
@@ -609,6 +609,13 @@ message ProviderConfig {
   }
   repeated ResourceConfig resources = 6;
   repeated string allowed_account_types = 7;
+  message ProviderParameter {
```

Comment:
@haveiss what will this be used for?

Reply:
Based on the issue and mas rahmat, it will be used whenever a new provider is registered via Guardian the custom question is defined for that during the creation itself.
Example use case for asking github username if the provider is github

CMIIW mas @rahmatrhd

Reply:
Yes, understood that. but this parameter is to collect questions.